models/evc.py

```diff
@@ -708,13 +708,25 @@ def remove_failover_flows(self, exclude_uni_switches=True,
         if sync:
             self.sync()
 
-    def remove_current_flows(self, force=True, sync=True):
+    def remove_current_flows(
+        self,
+        force=True,
+        sync=True,
+        return_path=False
```

Comment:
The `return_path=False` will be accepted in this code review. But, let's map a tech debt to be solved later:

All methods that remove or install flows should return a dataclass which will aggregate optional attributes such as old_path: dict[str, int], new_flows: dict[str, list[dict]], that way for all methods they return the same kind of data, and fields that doesn't apply is returned as empty. 

Often times methods with too many boolean args tends become difficult to reason, maintain and test, ours here is still maintainable and manageable. The next augmentation will need to cover that debt, tech, can you map this for us?

Reply:
[Issue](https://github.com/kytos-ng/mef_eline/issues/582) created